ImportError when importing `art 1.9.0` due to `matplotlib` in `laser_attack.utils`

**Describe the bug**
when I attempt to import `art` 1.9.0 I get a `ModuleNotFoundError`
```
../../../.cache/pypoetry/virtualenvs/neurips-2021-nd2ZNWoA-py3.7/lib/python3.7/site-packages/art/attacks/evasion/laser_attack/utils.py:30: in <module>
    import matplotlib.pyplot as plt
E   ModuleNotFoundError: No module named 'matplotlib'
```

I think this is due to the import of `matplotlib` in `laser_attack.utils` that @mateuszGorczany added last month?
https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/cbf4f9df679a9e434e3c42de89ed438400335d44/art/attacks/evasion/laser_attack/utils.py#L30

Looks like you all have it declared as an extras or dev dependency, but not in `install_requires`. I'm guessing it's not intended for import to fail just because I haven't installed extras though?

```
install_requires = [
    "numpy>=1.18.0",
    "scipy>=1.4.1",
    "scikit-learn>=0.22.2,<1.1.0",
    "six",
    "setuptools",
    "tqdm",
    "numba>=0.53.1",
]
```

**System information (please complete the following information):**
 - Ubuntu
 -  3.7
 -  1.9.0
 - torch 1.10

--- a/art/attacks/evasion/laser_attack/utils.py
+++ b/art/attacks/evasion/laser_attack/utils.py
@@ -27,7 +27,6 @@ import string
 from typing import Any, Callable, List, Tuple, Union
 
 import numpy as np
-import matplotlib.pyplot as plt
 
 
 class Line:
@@ -256,6 +255,8 @@ def save_nrgb_image(image: np.ndarray, number=0, name_length=5, directory="attac
     :param name_length: Length of the random string in the name.
     :param directory: Directory where images will be saved.
     """
+    import matplotlib.pyplot as plt
+
     alphabet = np.array(list(string.ascii_letters))
     Path(directory).mkdir(exist_ok=True)
     im_name = f"{directory}/{number}_{''.join(np.random.choice(alphabet, size=name_length))}.jpg"

